datamap/bloom/src/main/java/org/apache/carbondata/datamap/bloom/BloomCoarseGrainDataMapFactory.java

```diff
@@ -65,9 +65,18 @@
    * and all the indexed value is distinct.
    */
   private static final int DEFAULT_BLOOM_FILTER_SIZE = 32000 * 20;
+  /**
+   * property for fpp(false-positive-probability) of bloom filter
+   */
+  private static final String BLOOM_FPP = "bloom_fpp";
```

Comment:
Do user need to configure this? What is the relationship with DEFAULT_BLOOM_FILTER_SIZE

Reply:
yes, by setting n and p, user can control the bloomfiltersize (for memory) and the number of hash function(for cpu).

Reply:
Testcase will be added after your PR #2255 is merged. It makes lots of changes to the code.